spec/services/govspeak_document/payload_options_spec.rb

```diff
@@ -14,12 +14,42 @@
 
       payload_options = GovspeakDocument::PayloadOptions.new("govspeak", edition)
       actual_image_options = payload_options.to_h[:images].first
-      expect(actual_image_options[:alt_text]).to eq("Some alt text")
-      expect(actual_image_options[:caption]).to eq("An optional caption")
-      expect(actual_image_options[:credit]).to eq("An optional credit")
-      expect(actual_image_options[:url]).to match("filename.png")
-      expect(actual_image_options[:url]).to match("/media")
-      expect(actual_image_options[:id]).to eq("filename.png")
+
+      expect(actual_image_options).to match(
+        a_hash_including(
+          id: "filename.png",
+          alt_text: "Some alt text",
+          caption: "An optional caption",
+          credit: "An optional credit",
+          url: a_string_matching(%r{/media/.*/filename.png}),
+        ),
+      )
+    end
+
+    it "returns the remote asset options for a file attachment" do
```

Comment:
Just a minor thing that the it text reads differently to the one above - not really fussed which one we keep :-).

Reply:
I've made the assertions consistent now, thanks